prac_05/birthdays.py

```diff
@@ -0,0 +1,29 @@
+"""Program that takes names and birthdates as input and returns current age"""
+
+NUM_PEOPLE = 5
+
+def main():
+    ages = {}
+    longest_name = 0
+    for i in range(NUM_PEOPLE):
+        name = input("Enter person {}'s name: ".format(i + 1))
+        if len(name) > longest_name:
+            longest_name = len(name)
+        birthdate = input("Enter {}'s birthdate using dd/mm/yyyy: ".format(name))
+        day, month, year = birthdate.split("/")
+        if int(month) >= 9:
```

Comment:
This if and 2018 or 2019 does the job that `calculate_age` should do. Move this logic into the function and the main code will read better (better separation).